piet/src/text.rs

```diff
@@ -111,8 +111,8 @@ pub enum TextAttribute {
     Weight(FontWeight),
     /// The foreground color of the text.
     ForegroundColor(crate::Color),
-    /// Italics.
-    Italic(bool),
+    /// The [`FontStyle`](enum.FontStyle.html).
```

Comment:
A classic boilerplate comment. I'd probably add "; regular or italic." This will help people who are specifically searching for how to apply italics imho.